File: haushaltsbuch-api/src/main/java/com/github/haschi/haushaltsbuch/api/refaktorisiert/EröffnungsbilanzkontoErstellt.java
package com.github.haschi.haushaltsbuch.api.refaktorisiert;

import com.github.haschi.modeling.de.Ereignis;

import java.util.List;

@Ereignis
public interface EröffnungsbilanzkontoErstellt
{
    List<Buchung> buchungen();
}

File: haushaltsbuch/src/main/kotlin/org/github/haschi/haushaltsbuch/api/events/EröffnungsbilanzkontoErstellt.kt
package org.github.haschi.haushaltsbuch.api.events

import org.github.haschi.haushaltsbuch.core.Eröffnungsbilanzkonto

data class EröffnungsbilanzkontoErstellt(
        val eröffnungsbilanzkonto: Eröffnungsbilanzkonto)
